scripts/publish.js

```diff
@@ -13,8 +14,15 @@ exec(`npm version ${newVersionArg}`);
 // push up new version commit and tag
 exec("git push --follow-tags");
 
-// publish to public npm
-exec("npm publish");
+// use the readline module to simulate npm prompt
+const rl = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout,
+});
 
-// deploy storybook to github pages
-exec("npm run deploy-storybook");
+// prompt for otp and publish to public npm
+rl.question("Enter OTP for public npm:", (otp) => {
```

Comment:
Please add a space after the colon
```suggestion
rl.question("Enter OTP for public npm: ", (otp) => {
```